### PR TITLE
CVPN-791: Add documentation about wireshark and PMTU

### PR DESCRIPTION
**Pull request by kp-mariappan-ramasamy**
Add documentation about wireshark and PMTU

### CODE REVIEW

File: docs/pmtu_discovery.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+[`Packetization Layer Path MTU Discovery for Datagram Transports (RFC 8899)`](https://datatracker.ietf.org/doc/html/rfc8899)
+
+Please refer the above RFC for the state machine.
+
+The following are the things which is notable in Lightway's implementation:
+
+- `Lightway` uses Ping/Pong message with id != 0 as PLPMTU probe message
+- `Packetization Layer` defined in the RFC is `Lightway` itself (not including DTLS or lower layers)
+
+After calculating Path MTU, `Lightway` uses it for handling inside packets(from tunnel):
```
Comment (xv-ian-c) [inline docs/pmtu_discovery.md]: nit: missing space in "packets(from"

File: docs/pmtu_discovery.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+1. Update TCP MSS value in TCP SYN packets based on the PMTU value
+1. Fragment UDP packets inside lightway protocol if the size is larger than PMTU,
+   which will be reassembled at the other end.
```
Comment (xv-ian-c) [inline docs/pmtu_discovery.md]: Maybe include the names of the functions etc which are responsible for these two things?

File: docs/pmtu_discovery.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Path MTU Discovery
+
+`Lightway` protocol supports calculating the Path MTU between server and client dynamically.
+
+`Lightway` implements the following RFC to Path MTU calculation:
```
Comment (xv-ian-c) [inline docs/pmtu_discovery.md:5]: Is it worth including a reference to `lightway-core/src/connection/dplpmtud.rs` or `struct Dplpmtud` type?

File: docs/pmtu_discovery.md (hunk 2 above)
Comment (kp-mariappan-ramasamy) [reply docs/pmtu_discovery.md]: I added the DataFrag struct as reference

**Pull request merged**